CONTRIBUTING.md

```diff
@@ -44,7 +45,7 @@ docker build -t local-kolibri2zim .
 Scrape a channel (here we use the minimal channel, but you could use any other one of interest for your UI developments).
 
 ```
-docker run --rm -it -v $PWD/output:/output local-kolibri2zim kolibri2zim --name "minimal_test" --title "Minimal Kolibri Channel Test" --description "This is a minimal K
+docker run --rm -it -v "$PWD/output:/output" local-kolibri2zim kolibri2zim --name "minimal_test" --title "Minimal Kolibri Channel Test" --description "This is a minimal K
```

Comment:
I think `docker run --rm -it -v "$PWD/output":/output local-kolibri2zim ...` would be more correct, could you confirm it works as well?

Reply:
Yes, this also works fine. Will refactor this in the next commmit.